code/frameworks/vue3-webpack5/README.md

```diff
@@ -24,20 +24,22 @@ You can also build a [static version](https://storybook.js.org/docs/vue/sharing/
 
 ## Extending the Vue application
 
-Storybook creates a [Vue 3 application](https://v3.vuejs.org/api/application-api.html#application-api) for your component preview which can be imported as `import { app } from '@storybook/vue3'`.
-
+Storybook creates a [Vue 3 application](https://v3.vuejs.org/api/application-api.html#application-api) for your component preview.
 When using global custom components (`app.component`), directives (`app.directive`), extensions (`app.use`), or other application methods, you will need to configure those in the `./storybook/preview.js` file.
 
-For example:
+Therefore, Storybook provides you a a `setup` function from `@storybook/vue3`.
+This function receives a callback with the app instance of Storybooks Vue app passed that you can interact with.
```

Comment:
@d-koppenhagen, can you make the same adjustment as previously mentioned for the vue3-vite Readme when you're able?